homework2/task01/project1/task1.py

```diff
@@ -7,34 +7,30 @@
 4) Count every non ascii char
 5) Find most common non ascii char for document
 """
+import string
 from typing import List
 
 
 def file_presettings(file_path: str) -> str:
-    """
-    Return unpacked file, encode in readable format.
-
-    First sub() deletes '\n' between lines. Second sub() remove punctuation.
-    """
+    """Return unpacked file, encoded in readable format without punctuation."""
     with open(file_path, "r", encoding="unicode_escape") as f:
         text = f.read()
-        for p in '!"#$%&»«()*+,-./:;<=>?@_`{|}~':
+        for p in string.punctuation:
             if p in text:
                 text = text.replace(p, "")
         return text
 
 
-def get_unique_word_symbols(i: str) -> (str, str, int):
-    """Return key function for sorting."""
-    return len("".join(set(i))), i[0], len(i)
+def get_unique_word_symbols(i: str) -> (str, str):
+    """Return amount of unique symbols, first two letters for correct sequence sorting, also alphabetically."""
```

Comment:
> first two letters for correct sequence 

Looks like you tune your code to the specific test set.  Please make a general solution.